subprojects/tooling-api/src/main/java/org/gradle/tooling/events/configuration/ProjectConfigurationOperationResult.java

```diff
@@ -26,4 +30,33 @@
  */
 @Incubating
 public interface ProjectConfigurationOperationResult extends OperationResult {
+
+    /**
+     * Returns the results of plugins applied as part of the configuration of this project.
+     *
+     * <p>This may include plugins applied to other projects that are part of the current build,
+     * e.g. when using {@code subprojects {}} blocks in the build script of the root project.
+     */
+    List<? extends PluginConfigurationResult> getPluginConfigurationResults();
```

Comment:
Maybe `PluginApplication`?

Reply:
I agree that `PluginConfigurationResult` sounds a bit odd. In the case of cross-configuration, a plugin might have been applied multiple times and will only get a single result, though. And I'd like to include time spent in configuration callbacks in the measurement in a next step as well. Maybe the name isn't so bad after all, what do you think? 🤔

Reply:
I'll go ahead and merge this PR so I don't block myself. I'll ping you regarding the naming discussion. 🙂